src/client/components/base/Tile.tsx

```diff
@@ -12,18 +12,16 @@ interface TileProps {
 
 const Tile: React.FC<TileProps> = ({ className, children, href }) => {
   return (
-    <div
-      onClick={() => {
-        window.location.href = href;
-      }}
+    <a
+      href={href}
       className={classNames(
         'block bg-bg-accent shadow overflow-hidden border border-border',
```

Comment:
Since already explicitly block styled, the style is the same in the end even though anchors are normally inline. Used chrome dev tools to compare computed before and after, the computed before are all in the after